src/main/scala/bi/deep/DruidSchemaReader.scala

```diff
@@ -37,26 +38,24 @@ case class DruidSchemaReader(config: Config) extends DruidSegmentReader {
         sparkSchema
       }
     }
-    else {
-      sparkSchema
-    }
   }
 }
 
 object DruidSchemaReader {
-  def readSparkSchema(qisa: QueryableIndexStorageAdapter): SparkSchema = SchemaUtils.druidToSpark(readDruidSchema(qisa))
+  def readSparkSchema(queryableIndexStorageAdapter: QueryableIndexStorageAdapter):
+  SparkSchema = SchemaUtils.druidToSpark(readDruidSchema(queryableIndexStorageAdapter))
 
-  private def readDruidSchema(qisa: QueryableIndexStorageAdapter): DruidSchema = {
+  private def readDruidSchema(queryableIndexStorageAdapter: QueryableIndexStorageAdapter): DruidSchema = {
 
-    val metrics = qisa.getAvailableMetrics.asScala.toArray
-    val dimensions = qisa.getAvailableDimensions.iterator().asScala.toArray
+    val metrics = queryableIndexStorageAdapter.getAvailableMetrics.asScala.toArray
+    val dimensions = queryableIndexStorageAdapter.getAvailableDimensions.iterator().asScala.toArray
 
     val dCapabilities = dimensions.map { dimension =>
-      (dimension, qisa.getColumnCapabilities(dimension).asInstanceOf[ColumnCapabilitiesImpl])
+      (dimension, queryableIndexStorageAdapter.getColumnCapabilities(dimension).asInstanceOf[ColumnCapabilitiesImpl])
     }
 
     val mCapabilities = metrics.map { metric =>
-      (metric, qisa.getColumnCapabilities(metric).asInstanceOf[ColumnCapabilitiesImpl])
+      (metric, queryableIndexStorageAdapter.getColumnCapabilities(metric).asInstanceOf[ColumnCapabilitiesImpl])
```

Comment:
I think in this case we can simplify name like `queryableIndexStorageAdapter` to give more context.
Because it is just some OOP gibbrish. We don't care it is an adapter nor queryable. I would say let use name 
`druidStorage` then it is nice to read like:
- `druidStorage.getAvailableMetrics` 
- `druidStorage.getAvailableDimensions`